src/task/script.js

```diff
@@ -461,12 +480,34 @@ function setAssignee(assignee) {
 
 function clearSuggestionList() {
   const userNames = document.querySelectorAll('.list-item');
+  document.getElementById('suggested-users').classList.add('hidden');
   userNames.forEach((user) => {
     user.remove();
   });
 }
 
+function createSuggestedUserLists() {
+  const suggestedUsersDiv = document.getElementById('suggested-users');
+  const suggestedUsersContainer = document.getElementById(
+    'suggested-users-container',
+  );
+
+  suggestedUsersContainer.innerHTML = '';
+  suggestedUsersDiv.classList.remove('hidden');
+
+  suggestedUsers.forEach((user) => {
+    const listItem = document.createElement('p');
+    listItem.classList.add('list-item');
+    listItem.style.cursor = 'pointer';
```

Comment:
if you are adding the class for the tag then, can't you just add the style directly through the CSS file using this particular classname, as putting styles through javascript can be a computation expensive task

Reply:
done